history/characters/1_azerothian.txt

```diff
@@ -17177,4 +17178,110 @@
 		add_trait=lunatic
 	}
 	608.3.10={ death=yes }
+}
+
+#dynasty=413
+5816={
+	name=Isiah			#Reginald's greatgrandfather
+	dna="0cebc000ca0"
+	properties="ij0jgh0000000000000000"
+	properties="we0jga0000000000000000"
+	culture=azerothian religion=holy_light
+	martial=4 diplomacy=6 stewardship=3 intrigue=4 learning=4
+	495.10.8 = { birth=yes add_trait=creature_human }
+	522.7.13 = {
+		add_spouse=5821		#Reginald's grandmother
+	}
+	550.6.13={ death=yes }
+}
+5817={
+	name=Landen			#Reginald's grandfather
+	dynasty=413
+	dna="0lahj0f0c00"
+	properties="we0jga0000000000000000"
+	culture=azerothian religion=holy_light
+	martial=4 diplomacy=6 stewardship=3 intrigue=4 learning=4
+	father=5816		# Isiah
+	mother=5820		# Damia
+	523.10.8 = { birth=yes add_trait=creature_human }
+	546.7.13 = {
+		add_spouse=5821		#Reginald's grandmother
+	}
+	574.6.13={ death=yes }
+}
+5818={
+	name=Scot			#Reginald's father
+	dynasty=413
+	dna="000fe0f0d00"
+	properties="ei0efg0000000000000000"
+	culture=azerothian religion=holy_light
+	martial=5 diplomacy=4 stewardship=5 intrigue=3 learning=4
+	father=5817		# Landen
+	mother=5821		# Ysabel
+	547.3.20 = { birth=yes add_trait=creature_human }
+	569.4.16 = {
+		add_spouse=5822		#Reginald's mother
+	}
+	590.10.8={ death=yes }
+}
+5819={
+	name=Reginald
+	dynasty=413
+	dna="0clif0i0d00"
+	properties="bh0fda0000000000000000"
+	culture=azerothian religion=holy_light
+	martial=6 diplomacy=6 stewardship=4 intrigue=4 learning=4
+	add_trait=tough_soldier add_trait=robust add_trait=stubborn add_trait=honest add_trait=brave
+	father=5818		# Scot
+	mother=5822		# Jane
+	572.1.1 = { birth=yes add_trait=creature_human }
+	588.1.1 = { add_trait=class_warrior_6 }
+	590.1.1 = {
+		employer=8						# Varian Wrynn
+		effect = {
+			liege = { job_marshal = { leave_council_position_no_opinion_effect = yes } }
+			leave_council_position_no_opinion_effect = yes
+			give_job_title = job_marshal
+		}
+	}
+	660.1.1={ death=yes }
+}
+5820={
+	name=Damia				#Reginald's greatgrandmother
+	dna="000i00a0ca0"
+	properties="bk0e000000000000000000"
+	female=yes
+	culture=azerothian religion=holy_light
+	martial=1 diplomacy=6 stewardship=6 intrigue=5 learning=3
+	498.3.26 = { birth=yes add_trait=creature_human }
+	522.2.15 = {
+		add_spouse=5816		#Reginald's greatgrandfather
+	}
```

Comment:
Not needed if `5816` makes the character his spouse.

Reply:
thx, done